src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/template.json

```diff
@@ -353,7 +345,7 @@
       "dataType": "bool",
       "defaultValue": "true",
       "onlyIf": {
-        "wasm": "true"
+        "platforms": "wasm"
```

Comment:
This should include skia.gtk

Reply:
We can revisit this in another PR... it would seem to me that the display name should be changed then.